packages/next/build/utils.ts

```diff
@@ -511,7 +512,7 @@ export async function buildStaticPaths(
   const staticPathsResult = await unstable_getStaticPaths()
 
   const expectedReturnVal =
-    `Expected: { paths: [] }\n` +
+    `Expected: { paths: [], fallback: boolean }\n` +
     `See here for more info: https://err.sh/zeit/next.js/invalid-getstaticpaths-value`
```

Comment:
We should have a customized message for `fallback` I think 🤔

Reply:
https://github.com/zeit/next.js/pull/10701/files#diff-1628ba534c7c01cce2a072fdd334e974R542